Running-Locally.md

```diff
@@ -72,17 +72,18 @@ These environment variables can be placed in the `raincatcher-demo-cloud/Gruntfi
       FH_MBAAS_PROTOCOL: "<FH_MBAAS_PROTOCOL value>",
       FH_WIDGET: "<FH_WIDGET value>",
       FH_APP_API_KEY: "<FH_APP_API_KEY value>",
+
+      // This value can be found in `raincatcher-demo-cloud/Gruntfile.js`
+      // in the FH_SERVICE_MAP environment variable
+      WFM_AUTH_GUID: "iidn3tvprs62asdebat5m3eg"
     }
   }
   ...
 }
 ```
 
+## Run The Applications
 
-### Using A Local Service To Authenticate Users
```

Comment:
It's using RHMAP terminology but its essentially just another cloud app running locally. It can be run outside of the platform.